packages/chakra-ui-lib/src/components/Table/TableTd.tsx

```diff
@@ -44,15 +52,18 @@ export const TableTd: React.FC<{
     case 'button':
       const onClick = () => {
         onClickItem();
-        column.buttonConfig.handlers.forEach(handler => {
+        rawColumn.buttonConfig.handlers.forEach(handler => {
+          const evaledHandler = services.stateManager.deepEval(handler, true, {
+            [LIST_ITEM_EXP]: item,
+          });
           services.apiService.send('uiMethod', {
-            componentId: handler.componentId,
-            name: handler.method.name,
-            parameters: handler.method.parameters,
+            componentId: evaledHandler.componentId,
+            name: evaledHandler.method.name,
+            parameters: evaledHandler.method.parameters,
           });
         });
       };
-      content = <Button onClick={onClick}>{column.buttonConfig.text}</Button>;
+      content = <Button onClick={onClick}>{buttonConfig.text}</Button>;
```

Comment:
Not related to this PR: do we still need a build-in button integration when we support module renderer? Especially when the module renderer can accept any component as a module.

Reply:
I think there could be these reasons:
1. Module is not stable now.
2. Button in table is very common and build-in button is much more convinient than a module.
3. Create a module only for a button maybe too heavy.